pkg/tfbridge/schema.go

```diff
@@ -655,6 +650,18 @@ func (ctx *conversionContext) makeObjectTerraformInputs(
 		return nil, err
 	}
 
+	if tfs != nil && ctx.ApplyMaxItemsOneDefaults {
```

Comment:
As discussed this code is not enabled yet as nothing sets ctx.ApplyMaxItemsOneDefaults ; instead we're going to try to land cross-tests first to increase coverage in this area significantly before iterating further through downstream tests.